multicluster/config/certmanager/certificate.yaml

```diff
@@ -0,0 +1,25 @@
+# The following manifests contain a self-signed issuer CR and a certificate CR.
```

Comment:
similarly all the config yamls could be in multicluster/build/yamls/ to be consistent with parent repo

Reply:
Kube-builder specific.